### DISCUSSION TITLE
你為什麼總感到無聊

### DISCUSSION THREAD

**Discussion by tower20021202** [1101-GS4719-C/Python-Community-GS4719-C#1300] [General]
無論曾經多麽喜歡的電影、電視劇、遊戲和網購那些讓人開心的事情，如經變得越來越沒意思了，但習慣往往還是會讓我們一直重複他們，但每次做完，只有無聊還有浪費時間的愧疚感，常會想也許睡一覺隔天就好了。可是一起床，經歷了一天的工作和學習之後，又會把我們帶回上述無聊的循環當中。據統計我們每年都會有131天會感到無聊，然而我們現今明明有了越來越多的娛樂種類，但為何卻會越來越無聊呢？
原因其實是我們大腦會釋放一種物質較多巴胺，當遇到刺激時，大腦就會釋放多巴胺使你感到快樂，然而釋放完多巴胺的大腦隨後並不會直接回到基礎水平，而是會陷入一段低谷，伴隨著焦慮和無聊，但一段時間就會自然恢復基準水平，但若是在低谷時馬上進行會快速上升多巴胺的事，例如滑手機、打遊戲、吃甜食，但這會使大腦的多巴安峰值降低，隨後還會陷入更深的低谷，如此的惡性循環，最終陷入長期的抑鬱和無聊之中。這也是為什麼在世界最富裕的國家中，同時也有最高的自殺率、憂慮、抑鬱和身體上的疼痛。
因此除了能做能快速釋放多巴安的事情，我們應該還能做什麼來增加我們的快樂呢？

**Comment by poweichen00** [comment]
我覺得可以多投入在你有興趣的事情上，增加成就感，但我看你平常蠻快樂的，應該沒這問題。

**Comment by KevinCheng42** [comment]
多巴胺會讓我們的心情感到愉悅，根據網路上的資料，除了我們已知的娛樂活動可以快速刺激多巴胺的分泌外，適度的運動抑或是攝取充足的營養，甚至是靜坐冥想這類穩定增加多巴胺濃度的方法，不會像上述娛樂活動快速刺激多巴胺分泌，並有助於我們維持愉悅且「不易陷入更深的低谷」，使自己感到焦慮～

**Comment by CCello** [comment]
我認為要增加快樂感並不是只有從外部獲取快感，更重要的是從內部找到真正的快樂。透過建立良好的關係、增加身體活動量、學習新事物和培養興趣愛好等方式，我們可以從內部找到持久的快樂感。

**Comment by Jack-1D** [comment]
對我來說無聊可能來自於很多地方。像是沒事做，或是我很無聊時就會感到很無聊，又或是想要感覺無聊時，身體很自然就會感到無聊，也有可能是重複看到相同的文章，而讓我感到疲倦。
[https://www.youtube.com/watch?v=fOHdnxoscws&ab_channel=shuaisoserious](https://www.youtube.com/watch?v=fOHdnxoscws&ab_channel=shuaisoserious)

**Comment by YoYooHuang** [comment]
無聊對我來說是一種放鬆，但如果時常感到無聊，那應該認真思考一下自己到底想做什麼。其實我常常會發現，雖然事情很多，但如果我把一些“無聊”的時間拿來做一些有生產力的事情，一切就不會那麼緊湊。如果沒什麼急迫需要完成的工作的話，其實也不妨出去走走，打打球，散散步，也能放鬆自己的身心！

**Comment by orianna901001** [comment]
有時候無聊的時候，我覺得是時候挖掘一些平常不會做的事情，像我自己就會需要一些新鮮感或是刺激感，例如發現一個新的很好逛的地方。

**Comment by ziihui** [comment]
我認為做自己喜歡的事情可以讓生活更有趣、更有意義。
透過深入了解和學習，我們可以提升自己的技能和自信心，在過程中還能遇到更多志同道合的朋友。更重要的是，培養興趣還有助於減壓、舒緩焦慮，提升心理健康！

**Comment by Evening218** [comment]
我認為如果連做自己喜歡的事時都會感到無聊,那麼可以試試和別人一起做,或著是嘗試家人朋友們的興趣,和別人相處時因為思維方式和習慣的不同,應該比較能夠帶給我們新的刺激

**Comment by RogelioKG** [comment]
老實說，自從開始規劃自己的學習計畫後，
無聊的時間越來越少了，而腦袋被很多學習進度佔線。
其實我也有在想，暑假的自學時間是不是反而不應該排那麼滿比較好。
或許我應該空點時間讓自己的腦袋放空（多一點無聊的時間）。
總覺得腦袋時常佔線，反而會在處理生活其他事務上變得笨拙。

**Comment by Aaron0810** [comment]
我自己是比較少感到無聊的，可能因為平常生活相對比較忙碌一點? 這樣有個好處是因為你的娛樂活動時間不會太多，所以不會對娛樂活動感到厭煩，也因此在少有的娛樂活動時間中能真正的娛樂到，也就不會感到無聊了。

**Comment by Elaine2899** [comment]
我曾經看過一個言論是，如果我們一職處在情緒高峰，那麼我們將慢慢減緩對情緒高峰的感受，對快樂也是。我們的情緒需要呈現正常波動狀才會正常感受快樂。因此我們的生活需要低潮才能感受到快樂！我個人也非常認同這樣的觀點。

**Comment by pudding0803** [comment]
雖然玩遊戲之類的可能會讓心情變好，但如果每天持續一直玩的話，相信到最後也會感到空虛、乏味。因此我們或許可以每天嘗試不同的事物，嘗試不同的興趣，不讓生活變得過於重複，避免習慣了之後就不會感到快樂

**Comment by PaulSu0905** [comment]
我覺得隨著年紀的增長，確實更常有無聊的感覺，對於本來有興趣的事物感到乏味的速度也變得更快了，但我也發現雖然自己可能恨無聊，但如果有朋友夠一起，就可以把無聊的事變得有趣，所以不訪嘗試多與能讓自己快樂的朋友相處。
